app/policies/idv/gpo_verify_by_mail_policy.rb

```diff
@@ -37,7 +37,7 @@ def profile_too_old?
 
     private
 
-    def disabled_for_biometric_comparison?
+    def disabled_for_facial_match?
```

Comment:
not part of the DB table but probably fine to fix anyways

Reply:
yeah sorry i just snuck this in because it somehow avoided being committed in the first big PR